src/commands/up.rs

```diff
@@ -85,6 +85,12 @@ pub struct UpCommand {
     #[clap(long, takes_value = false)]
     pub direct_mounts: bool,
 
+    /// For local apps, specifies to perform `spin build` before running the application.
+    ///
+    /// This is ignored on remote applications, as they are already built.
+    #[clap(long, takes_value = false, env = "SPIN_BUILD_BEFORE_UP")]
```

Comment:
might be nice to make this env var more generic in case we want to use it for other commands like deploy in the future.
`SPIN_ALWAYS_BUILD` wdyt?

Reply:
@michelleN That was my first thought too!  It would be spot on for `.profile` and could be shared with `spin cloud deploy`.  But I got cold feet thinking "it looks silly writing `SPIN_ALWAYS_BUILD=true spin up`".  BUT NOBODY WOULD EVER WRITE THAT IVAN.  Right? ...right?